packages/ui/src/common/components/Pagination/Pagination.tsx

```diff
@@ -11,27 +11,33 @@ interface PaginationProps {
   page?: number
 }
 
-export const Pagination: FC<PaginationProps> = ({ pageCount, handlePageChange, page }) => (
-  <StyledPaginateContainer>
-    <ReactPaginate
-      pageCount={pageCount}
-      marginPagesDisplayed={3}
-      pageRangeDisplayed={3}
-      containerClassName="pagination"
-      pageLinkClassName="pagination__link"
-      pageClassName="page"
-      activeClassName="page--active"
-      activeLinkClassName="pagination__link--active"
-      breakLinkClassName="pagination__link"
-      previousLabel={<Arrow direction="left" />}
-      nextLabel={<Arrow direction="right" />}
-      nextLinkClassName="pagination__link"
-      previousLinkClassName="pagination__link pagination__link--previous"
-      onPageChange={(value) => handlePageChange(value.selected + 1)}
-      forcePage={page && page - 1}
-    />
-  </StyledPaginateContainer>
-)
+export const Pagination: FC<PaginationProps> = ({ pageCount, handlePageChange, page }) => {
+  if (pageCount < 2) {
```

Comment:
```suggestion
  if (pageCount ?? 0 < 2) {
```
This way we could catch nullish pageCount in here instead of parent component, making it a bit more concise.
I don't have a strong opinion on this though, can be left as is.

Reply:
@p-sad Than is loading also redundant, because pageCount also will be undefined/null